README.md

```diff
@@ -23,6 +23,19 @@ work in progress.
 Please file reports on the
 [GitHub Issue Tracker](https://github.com/dart-lang/dart-pad/issues).
 
+## Running locally
+
+The project contains a tiny appengine for redirecting to gists and the mobile UI. To run this:
+- Download the AppEngine SDK for Python here: https://cloud.google.com/appengine/downloads
+- Navigate to the root of the checkout folder
+- run `pub get`
+- run `dart tool/grind.dart build`
```

Comment:
Before that you need to do: sudo npm install -g bower
Which means you may need to install node.js first.
Running the command Luke used, I got this at the end:

  build/web/mobile.html original: 6k
  vulcanize --strip --output mobile.html mobile.html

ProcessException: No such file or directory
  Command: vulcanize --strip --output mobile.html mobile.html
#0      _ProcessImpl._runAndWait (dart:io-patch/process_patch.dart:474)
#1      _runNonInteractiveProcessSync (dart:io-patch/process_patch.dart:626)
#2      Process.runSync (dart:io-patch/process_patch.dart:64)
#3      run (package:grinder/src/run.dart:26:34)
#4      build (file:///Users/messick/src/editors/dart-pad-master/tool/grind.dart:62:3)